Controller/FormBuilderAdminController.php

```diff
@@ -2,8 +2,77 @@
 
 namespace Pirastru\FormBuilderBundle\Controller;
 
+use Pirastru\FormBuilderBundle\Entity\FormBuilder;
+use Pirastru\FormBuilderBundle\Entity\FormBuilderSubmission;
 use Sonata\AdminBundle\Controller\CRUDController;
+use Sonata\AdminBundle\Exception\ModelManagerException;
+use Sonata\AdminBundle\Exception\ModelManagerThrowable;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Request;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Entity;
 
 class FormBuilderAdminController extends CRUDController
 {
+    #[Route(path: "/submissions/{submission_id}/delete", name: "admin_pirastru_formbuilder_formbuilder_submission_delete", methods: "POST")]
```

Comment:
Route configuration is not needed here. This is done in  the`configureRoutes` method of your admin class.